core/services/job/spawner.go

```diff
@@ -137,11 +138,12 @@ func (js *spawner) stopService(jobID int32) {
 		service := aj.services[i]
 		err := service.Close()
 		if err != nil {
-			js.lggr.Errorw("Error stopping job service", "jobID", jobID, "error", err, "subservice", i, "serviceType", reflect.TypeOf(service))
+			js.lggr.Criticalw("Error stopping job service", "jobID", jobID, "error", err, "subservice", i, "serviceType", reflect.TypeOf(service))
 		} else {
-			js.lggr.Infow("Stopped job service", "jobID", jobID, "subservice", i, "serviceType", reflect.TypeOf(service))
+			js.lggr.Debugw("Stopped job service", "jobID", jobID, "subservice", i, "serviceType", reflect.TypeOf(service))
```

Comment:
I wonder if this is more efficient, since we only need a string?
```suggestion
			js.lggr.Debugw("Stopped job service", "jobID", jobID, "subservice", i, "serviceType", fmt.Sprintf("%T", service))
```